automatic/dropbox/dropbox.nuspec

```diff
@@ -6,11 +6,14 @@
     <title>Dropbox</title>
     <authors>Dropbox</authors>
     <owners>chocolatey,ferventcoder,the-running-dev</owners>
-    <licenseUrl>http://www.dropbox.com/terms</licenseUrl>
+    <licenseUrl>https://www.dropbox.com/terms</licenseUrl>
     <projectUrl>https://www.dropbox.com/</projectUrl>
-    <iconUrl>https://raw.githubusercontent.com/chocolatey/chocolatey-coreteampackages/master/icons/dropbox.svg</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/9d90583a4103d4a9db86425a5fbda79cc845b4ef/icons/dropbox.svg</iconUrl>
+    <mailingListUrl>https://www.dropboxforum.com/</mailingListUrl>
+    <docsUrl>https://www.dropbox.com/help</docsUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>
+    ## Features
```

Comment:
Sorry, that's not I meant when I said to add a feature section.
I meant you should add a features section after the current description.
Like what is done with the [qbittorrent](https://chocolatey.org/packages/qbittorrent) package (and most other packages in this repo)